src/AppBundle/GraphQL/Mutation/CharacterMutation.php

```diff
@@ -36,7 +39,8 @@ function createCharacter(string $userId, string $characterName)
         $this->game->getEntityManager()->flush();
 
         return [
-            "character" => new CharacterType($this->getGame(), $character)
+            "characterEdge" => CharacterConnection::createEdgeFor($user, new CharacterType($this->getGame(), $character)),
+            "user" => new UserType($this->getGame(), $user),
```

Comment:
Relay asks specifically for this field during the mutation which it probably wouldn't if it isn't required.